src/Crypto/RNG.hs

```diff
@@ -139,9 +139,13 @@ type InnerCryptoRNGT = ReaderT CryptoRNGState
 
 -- | Monad transformer with RNG state.
 newtype CryptoRNGT m a = CryptoRNGT { unCryptoRNGT :: InnerCryptoRNGT m a }
-  deriving ( Alternative, Applicative, Functor, Monad
-           , MonadBase b, MonadCatch, MonadError e, MonadIO, MonadMask, MonadPlus
-           , MonadThrow, MonadTrans )
+  deriving ( Alternative, Applicative, Functor, Monad, MonadBase b
+           , MonadCatch, MonadError e, MonadIO, MonadMask, MonadPlus
+           , MonadThrow, MonadTrans
+#if MIN_VERSION_base(4,13,0)
```

Comment:
Do we need the ifdef? MonadFail is in base since 4.9.0.0

Reply:
I thought we want to support older `base` as well, so I just used the migration guide for GHC-8.8. But I can remove it if we want to support only GHC-8 and later.

Reply:
Yes please, there is no point supporting older GHC versions, 8.0 is almost 4 years old.

Reply:
Done